src/views/credits/credits.jsx

```diff
@@ -10,9 +12,9 @@ var Credits = React.createClass({
     render: function () {
         return (
             <div className="inner credits">
-                <h1>Scratch Credits and Contributors</h1>
-                <h2>MIT Scratch Team</h2>
```

Comment:
One more here to remove from the json, since `MIT Scratch Team` is a proper noun :)